docs/docs/adding-forms.md

```diff
@@ -1,35 +1,35 @@
 ---
-title: Adding Forms
+title: फॉर्म को जोड़ना
 ---
 
-Gatsby is built on top of React. So anything that is possible with a React form is possible in Gatsby. Additional details about how to create React forms can be found in the [React forms documentation](https://reactjs.org/docs/forms.html) (which happens to be built with Gatsby!)
+Gatsby के ऊपर बनाया गया है React. तो सब कुछ जो संभव है React फॉर्म में ,वो संभव है Gatsby में भी. अतिरिक्त जानकारि की कैसे बनाएं React फॉर्म ,यहाँ पाया जा सकता है [React forms documentation](https://reactjs.org/docs/forms.html) (जो बनाया गया है Gatsby! से)
```

Comment:
`React.` => `React।`

Reply:
`React|` => `React।`

use the one i have suggested. copy paste it.

Reply:
this is still not fixed. you can see the difference in below image

<img width="150" alt="Screen Shot 2020-05-27 at 10 37 28 AM" src="https://user-images.githubusercontent.com/4654382/82980041-1d4c2980-a006-11ea-97ba-c8ad872ac1d1.png">